README.md

```diff
@@ -30,23 +39,43 @@ Build binaries automatically using the instructions from the Makefile: `make all
 
 This command:
 * Generates three configuration files (`migrate.yaml`, `observer.yaml`, `observerapi.yaml`) and places them into the hidden `./.artifacts` directory.
-* Generates thee binaries (`migrate`, `observer`, `api`) and places them into `./cmd/migrate/*.go`, `cmd/observer/*.go`, `./cmd/api/*.go` respectively.
+* Generates thee binaries (`migrate`, `observer`, `api`) and places them into `./.bin/migrate/*.go`, `./.bin/observer/*.go`, `./bin/api/*.go` respectively.
```

Comment:
Change the '*.go' to actual binaries names. They don't end with the '.go'

Reply:
fixed

Reply:
dunno how I overlooked it. it was plain stupid of me.